tests/compile-fail/data_race/dealloc_read_race_stack.rs

```diff
@@ -1,5 +1,6 @@
 // ignore-windows: Concurrency on Windows is not supported yet.
-// compile-flags: -Zmiri-disable-isolation
+// compile-flags: -Zmiri-disable-isolation -Zmir-opt-level=0
+// With optimizations (in particular #78360), the span becomes much worse, so we disable them.
```

Comment:
this one isn't using the error pattern scheme

Reply:
Yeah that was deliberate, to keep the test more readable. I am not set of getting 100% coverage with mir-opt-level=3, just some basic idea of what is happening.